dd-java-agent/agent-ci-visibility/src/main/java/datadog/trace/civisibility/domain/manualapi/ManualApiTestModule.java

```diff
@@ -79,7 +80,7 @@ public TestSuiteImpl testSuiteStart(
         linesResolver,
         coverageStoreFactory,
         executionResults,
-        null,
+        Collections.EMPTY_MAP,
```

Comment:
`Collections.EMPTY_MAP` is a raw type (there's no generics), so using it generates a compiler warning. It is better to use the corresponding method
```suggestion
        Collections.emptyMap(),
```